catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/AbstractCswSource.java

```diff
@@ -1366,6 +1368,10 @@ protected CapabilitiesType getCapabilities() {
         return caps;
     }
 
+    protected Subject getSystemSubject(org.codice.ddf.security.common.Security security) {
+        return org.codice.ddf.security.common.Security.runAsAdmin(security::getSystemSubject);
```

Comment:
It doesn't look like you're using the security instances in the callers except to pass to this method. Could refactor and get the instance here, where you need it.